lib/audited/auditor.rb

```diff
@@ -246,6 +248,10 @@ def auditing_enabled
       def auditing_enabled=(val)
         self.class.auditing_enabled = val
       end
+
+      def skip_writing?
+        Audited.only_authenticated_user ? ::Audited.store[:current_controller]&.send(Audited.current_user_method).blank? : false
```

Comment:
The `&.` operator is only available on Ruby 2.3 and higher, please check if the value's nil for compatibility with other versions.